omp/__init__.py

```diff
@@ -25,6 +26,26 @@ class OpenMP(object):
     """
 
     def __init__(self):
+        if c_compiler is None or c_compiler.compiler_type != 'msvc':
+            self.init_not_msvc()
+        else:
+            self.init_msvc()
+
+    def init_msvc(self):
+        vcomp_path = None
+        for i in range(15, 8, -1):
```

Comment:
This scanning looks very unrliable in the future....

Reply:
Yes, I need to fond a better way

Reply:
something like that: https://github.com/mhammond/pywin32/blob/master/setup.py#L1060 ?

Reply:
That also sounds super hacky. I prefer your scanning after all, but maybe trying a given version number based on the compiler path first?

Reply:
I can try to use compiler.__version or msvc_runtime_major from misc_util (numpy)?